### DISCUSSION TITLE
Where is the snapcraft.yaml now?

### DISCUSSION THREAD

**Discussion by jcrben** [dbeaver/dbeaver#32345] [DBeaver Development] [answered]
The snapcraft.yaml was added per this issue: https://github.com/dbeaver/dbeaver/issues/7988

@serge-rider made the still valid point that "external PRs may be useful". I want to review it to understand how it's set up a bit better.

But then it was removed in [this commit](https://github.com/dbeaver/dbeaver/commit/b5e1660f021674de3ec47413e7d2c47e813b9bc0) as "legacy" - where is it now? 

cc: @merlijn-sebrechts @riednyko

[orgwide search for snapcraft.yaml](https://github.com/search?q=org%3Adbeaver+snapcraft.yaml&type=issues)

**Comment by serge-rider** [comment]
Goo point. 
We are going to return to the main repo

**Comment by serge-rider** [comment, marked as answer]
It is back in https://github.com/dbeaver/dbeaver/tree/devel/product/snap
